sources/excel-basic/converter.rb

```diff
@@ -64,6 +64,10 @@ def to_coworker(row)
             coworker.email = row['Email']
         end
 
+        coworker.direct_phone_number = row['DirectPhoneNumber']
+        coworker.mobile_phone_number = row['MobilePhoneNumber']
+        coworker.home_phone_number = row['HomePhoneNumber']
```

Comment:
We should use the phone_helper to convert strings to phone numbers to avoid adding crap to the database.